Samples/MediaPlayerCS/MainPage.xaml.cs

```diff
@@ -195,6 +168,46 @@ private async void ExtractFrame(object sender, RoutedEventArgs e)
             }
         }
 
+        private async void LoadSubtitleFile(object sender, RoutedEventArgs e)
+        {
+            if (playbackItem != null)
+            {
+                FileOpenPicker filePicker = new FileOpenPicker();
+                filePicker.ViewMode = PickerViewMode.Thumbnail;
+                filePicker.SuggestedStartLocation = PickerLocationId.VideosLibrary;
+                filePicker.FileTypeFilter.Add("*");
```

Comment:
maybe the filter type should be limited to all those subtitle file types which are supported by uwp apis ?

Reply:
Do you have such a list? There is none available from docs afaik.

Reply:
This is just a sample, it shows how to do things in a very basic way. For a real app, this needs to be done differently anyways. Like, you'd have to provide the possibility to also select two files, for case of image based subtitles, and use different APIs to create the TimedTextSource. I do not plan to add all that in the sample.

Reply:
I agree, this doesn't have to be production ready code. You might even want to load the subtitle automatically on media open. Or have custom pickers.